libs/ledger-live-common/src/wallet-api/react.ts

```diff
@@ -837,11 +855,10 @@ export function useRecentlyUsed(
       recentlyUsed
         .map(r => {
           const res = manifests.find(m => m.id === r.id);
-          const distance = formatDistanceToNow(new Date(r.usedAt));
           return res
             ? {
                 ...res,
-                usedAt: distance[0].toUpperCase() + distance.slice(1) + " ago",
+                usedAt: calculateTimeDiff(r.usedAt),
```

Comment:
Did we check if `usedAt` is used somewhere else ? I suppose it shouldn't as it's only used in discover but better safe than sorry

Reply:
checked